publish-to-s3/action.yml

```
@@ -71,11 +74,12 @@ runs:
         set -f
 
         if [[ ${ONLY_LOG_ERRORS} == "true" ]]; then LOG_LEVEL="--only-show-errors "; fi
+        if [[ ${PUBLIC_READ} == "true" ]]; then ACCESS_PERMISSIONS="--acl public-read ";fi
```

Pretty sure the default for `ACCESS_PERMISSIONS` when embedded in a string should be the empty string, but if I'm misunderstanding how this default behaviour works for LOG_LEVEL above, I can also explicitly assign a default.

---

Yep should be fine, bash variables default to the empty string.

Nit:

```suggestion
        if [[ ${PUBLIC_READ} == "true" ]]; then ACCESS_PERMISSIONS="--acl public-read "; fi
```